Assembler.Base/Releasing/PositionCountValidators/BaseAssemblingPositionCountValidator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.RawAssemblingEntities;
+using Microsoft.Extensions.Logging;
+
+namespace Assembler.Base.Releasing.PositionCountValidators
+{
+    public abstract class BaseAssemblingPositionCountValidator : IValidator<RawMessageInAssembly>
+    {
+        private readonly int _minimumFramesCount;
+        private readonly ILogger _logger;
+
+        protected BaseAssemblingPositionCountValidator(int minimumFramesCount, ILoggerFactory loggerFactory)
+        {
+            _minimumFramesCount = minimumFramesCount;
+            _logger = loggerFactory.CreateLogger(ToString());
+        }
+
+        public bool IsValid(RawMessageInAssembly message)
+        {
+            var framesList = GetFramesList(message).ToArray();
```

Comment:
Instead of .ToArray() run .Count()

Reply:
I like it better this way, single enumeration